README.md

```diff
@@ -82,11 +82,26 @@ DB=<mysql|postgres|sqlite> yarn dev-server:start
 ```
 If you do not specify the `DB` argument, it will default to "mysql".
 
-### 6. Launch the admin ui
+### 6. Testing your changes locally
+This example shows how to test changes to the `core` package locally, but it will also work for other packages.
+1. Open 2 terminal windows: 
+- Terminal 1 for watching and compiling the changes of the package you are developing
+- Terminal 2 for running the dev-server 
+```
+# Terminal 1
+cd packages/core
+yarn watch
```

Comment:
Changes to GraphQL APIs & shared code in the /common package will not get picked up by this watch. Therefore there is a `watch:common-core` script in the root package.json which will watch both. That's the one I use for core dev.

Reply:
Ahh oke, so for changes to core we use `watch:core-common` and for all other packages we can do `cd packages/<package> && yarn watch` ?